tests/frameworks/test_fastapi.py

```diff
@@ -503,3 +503,54 @@ def test_response_header_capture(self):
         self.assertEqual("this too", asgi_span.data["http"]["header"]["X-Capture-This-Too"])
         self.assertIn("X-Capture-That-Too", asgi_span.data["http"]["header"])
         self.assertEqual("that too", asgi_span.data["http"]["header"]["X-Capture-That-Too"])
+
+    def test_non_async_function(self):
+        with tracer.start_active_span("test"):
```

Comment:
While the usage of the normal `tracer` instead of `async_tracer` is not new to this file,
since the `fastapi` instrumentation uses the `async_tracer`, I feel we should use `async_tracer` in the tests as well.